packages/celotool/src/lib/migration-utils.ts

```diff
@@ -36,7 +36,7 @@ function getAttestationKeys() {
 export function migrationOverrides() {
   const mnemonic = fetchEnv(envVar.MNEMONIC)
   const faucetedAccountAddresses = getFaucetedAccounts(mnemonic).map((account) => account.address)
-  const attestationBotAddresses = getAddressesFor(AccountType.ATTESTATION_BOT, mnemonic, 1)
+  const attestationBotAddresses = getAddressesFor(AccountType.ATTESTATION_BOT, mnemonic, 10)
```

Comment:
Could this be configurable?
```suggestion
  const attestationBotAddresses = getAddressesFor(AccountType.ATTESTATION_BOT, mnemonic, parseInt(fetchEnvOrFallback(envVar.ATTESTATION_BOTS, '10'), 10))
```